virt/virt-2-4-release-notes.adoc

```diff
@@ -132,20 +127,11 @@ than a unique one from a pool. Although rare, randomly assigned MAC addresses ca
 from the CLI by running the following `oc` patch command:
 +
 ----
-$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.3 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
+$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.4 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
 ----
 +
-This command points your subscription to upgrade channel `2.3` and enables automatic updates.
-
-* If you have {VirtProductName} 2.1.0 deployed, you must first upgrade {VirtProductName}.
-//Deleted link:https://bugzilla.redhat.com/show_bug.cgi?id=1817394[*BZ#1817394*] - It was closed.
-//Don't remove: this BZ is probably true fo all 2.x releases.
-** The `masquerade` binding method for virtual machines cannot be used in clusters with RHEL 7 compute nodes.
```

Comment:
This and the comment above look like a piece of a bug from 2.2 and not required for 2.4